examples/src/components/AttributeFilterComponentExample.jsx

```diff
@@ -15,9 +15,26 @@ export class AttributeFilterComponentExample extends Component {
     render() {
         return (
             <div>
+                <div>attribute defined by identifier</div>
                 <AttributeFilter
+                    projectId={projectId}
                     identifier={employeeNameIdentifier}
+                    fullscreenOnMobile={false}
```

Comment:
I think this prop should be documented separately and I would exclude false value here as it is default.

Reply:
done. Documentation in Properties table for Attribute Filter component in SDK UI doc